src/event/unflatten.rs

```diff
@@ -20,7 +20,7 @@ enum MapValue {
 
 #[derive(Debug, Clone, PartialEq)]
 pub struct Unflatten {
-    map: HashMap<Atom, MapValue>,
+    pub map: HashMap<Atom, MapValue>,
```

Comment:
I want to say this should be an iterator of `Iterator<Item = (Atom, MapValue)>`. I don't think we want to expose the internal implementation of this but instead can expose fns to get that data.

Reply:
I made `map` private again and added a `get` method for `Unflatten` instead.